### PR TITLE
Use Pact to test API endpoints

### PR DESCRIPTION
**Pull request by jaucourt**
https://eaflood.atlassian.net/browse/PSD-707

Initial contract tests for service endpoints

### CODE REVIEW

File: test/contract/ffc-demo-payment-service.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.start()
+  })
+
+  test('validates the expectations of ffc-demo-payment-web', async (done) => {
+    const opts = {
+      providerBaseUrl: 'http://localhost:3004',
```
Comment (johnwatson484) [inline test/contract/ffc-demo-payment-service.test.js:25]: Might be a good idea to source the port from the `config`?  Then if it changes we don't need to update the hard coded value in the test.

File: package.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+  "version": "2.4.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "build:watch": "nodemon --exec npm run build --ext scss --legacy-watch",
     "migrate": "sequelize db:migrate",
     "pretest": "npm run test:lint",
-    "start:watch": "nodemon --inspect=0.0.0.0 --ext js --legacy-watch index.js",
-    "test": "npm run test:unit",
+    "test": "jest --forceExit --detectOpenHandles",
```
Comment (spattisson) [inline package.json:10]: Do we want to remove the --forceExit here
```suggestion
    "test": "jest --detectOpenHandles",
```

Comment (spattisson) [reply package.json:10]: John's addressing this in his PR

File: test/contract/ffc-demo-payment-service.test.js (hunk 1 above)
Comment (spattisson) [reply test/contract/ffc-demo-payment-service.test.js]: Resolved by push a696b12

**Pull request merged**